gramhopper/configuration/globals_dict.py

```diff
@@ -0,0 +1,5 @@
+from typing import Dict
+from .trigger_response import TriggerResponse
+
+
+GlobalsDict = Dict[str, TriggerResponse]
```

Comment:
was it pylint's suggestion?

Reply:
Not exactly.
Using this type without naming it caused very long lines. If I'm not wrong, naming it in the original file caused dependency problems (maybe my memory just fools me).

Reply:
k, fine with me